src/components/Priorauth/PriorAuth.jsx

```diff
@@ -0,0 +1,370 @@
+import React, { Component } from "react";
+import { hot } from "react-hot-loader";
+import "./PriorAuth.css";
+
+class PriorAuth extends Component {
```

Comment:
This isn't an issue with your PR but we should really decide on a convention for class names, right now theres a mix between exporting the class at the top and using the hot loader, which we shouldn't need.